fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -1736,6 +1736,7 @@ expression
     | NOT expression                                                                      #logicalNot
     | left=expression operator=(AND|LOGICAL_AND) right=expression                         #logicalBinary
     | left=expression operator=(OR|LOGICAL_OR) right=expression                           #logicalBinary
+    | key=expression ':' value=expression                                                 #mapExpression
```

Comment:
```suggestion
    | key=expression ':' value=expression                                                 #mapExpression
```
don't add in here, it's will support sql like `where a:b = 1`?

Reply:
change it by introducing `mapExpressionList`